packages/client/hmi-client/src/workflow/ops/optimize-ciemss/tera-optimize-ciemss.vue

```diff
@@ -0,0 +1,729 @@
+						</div>
+						<div class="label-and-input">
+							<label for="time-unit">Unit</label>
+							<Dropdown
+								disabled
+								class="p-inputtext-sm"
+								:options="['Days', 'Hours', 'Minutes', 'Seconds']"
+								v-model="knobs.timeUnit"
+								placeholder="Select"
+							/>
+						</div>
+					</div>
+					<div>
+						<Button
+							v-if="showAdditionalOptions"
+							class="p-button-sm p-button-text"
+							label="Hide additional options"
+							@click="toggleAdditonalOptions"
+						/>
+						<Button
+							v-if="!showAdditionalOptions"
+							class="p-button-sm p-button-text"
+							label="Show additional options"
+							@click="toggleAdditonalOptions"
+						/>
+					</div>
+					<div v-if="showAdditionalOptions" class="input-row">
+						<div class="label-and-input">
+							<label for="num-samples">Number of stochastic samples</label>
+							<div class="input-and-slider">
+								<InputNumber
+									class="p-inputtext-sm"
+									inputId="integeronly"
+									v-model="knobs.numStochasticSamples"
+								/>
+								<Slider v-model="knobs.numStochasticSamples" :min="1" :max="100" :step="1" />
+							</div>
+						</div>
+						<div class="label-and-input">
+							<label for="solver-method">Solver method</label>
+							<Dropdown
+								disabled
+								class="p-inputtext-sm"
+								:options="['dopri5', 'euler']"
+								v-model="knobs.solverMethod"
+								placeholder="Select"
+							/>
+						</div>
+						<div class="label-and-input">
+							<!-- TODO: This could likely be better explained to user -->
+							<label> Minimized</label>
+							<Dropdown
+								class="toolbar-button"
+								v-model="knobs.isMinimized"
+								:options="[true, false]"
+							/>
+						</div>
+					</div>
+				</div>
+				<div class="form-section">
+					<h4>Intervention policy</h4>
+					<tera-intervention-policy-group-form
+						v-for="(cfg, idx) in props.node.state.interventionPolicyGroups"
+						:key="idx"
+						:config="cfg"
+						:parameter-options="modelParameterOptions.map((ele) => ele.id)"
+						@update-self="(config) => updateInterventionPolicyGroupForm(idx, config)"
+						@delete-self="() => deleteInterverntionPolicyGroupForm(idx)"
+					/>
+					<div>
+						<Button
+							icon="pi pi-plus"
+							class="p-button-sm p-button-text"
+							label="Add more interventions"
+							@click="addInterventionPolicyGroupForm"
+						/>
+					</div>
+				</div>
+				<div class="form-section">
+					<h4>Constraint</h4>
+					<div class="constraint-row">
+						<div class="label-and-input">
+							<label for="target-variable">Target-variable(s)</label>
+							<MultiSelect
+								class="p-inputtext-sm"
+								:options="modelStateOptions.map((ele) => ele.id)"
+								v-model="knobs.targetVariables"
+								placeholder="Select"
+							/>
+						</div>
+						<div class="label-and-input">
+							<label for="statistic">Statistic</label>
```

Comment:
Why do you have so many `<label for="..."` declarations? 
Are they suppose to be pointing to something?

Reply:
Not for anything, relic of Jami's initial work on this I will remove these